src/components/molecular/LabeledInput.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div :class="getClass">
+    <Paragraph>{{ label }}:</Paragraph>
+    <Input
+      small
+      :size="size"
+      :placeholder="placeholder"
+      :value="defaultValue"
+      @input="onInput"
+    />
```

Comment:
Since a label is available here, why not set `aria-label` on the Input? :thinking: 
Or use html-native `<label>` instead of / wrapping the `<Paragraph>` or its contents.

Reply:
The component now uses an html label and binds the label to the input 73d1734